api/samples/opcode_count.cpp

```diff
@@ -94,7 +94,9 @@ event_opcode_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *i
         static_cast<dr_spill_slot_t>(SPILL_SLOT_MAX + 1),
         IF_AARCHXX_(static_cast<dr_spill_slot_t>(SPILL_SLOT_MAX + 1)) &
             global_opcode_count,
-        1, DRX_COUNTER_LOCK);
+        1,
+        /* DRX_COUNTER_LOCK is not yet supported on ARM */
+        IF_X86_ELSE(DRX_COUNTER_LOCK, 0));
```

Comment:
So this will be a racy counter then?  Should that be acknowledged somewhere?

Reply:
If being racy is not good, I would expect a TODO or at least XXX comment or sthg complaining.

Reply:
Done.